build/linux/installer/scripts/livenessprobe.sh

```diff
@@ -26,6 +26,15 @@ then
  exit 1
 fi
 
+#test to exit non zero value if telegraf is not running
+(ps -ef | grep telegraf | grep -v "grep")
+if [ $? -ne 0 ]
+then
+ echo "Telegraf is not running" > /dev/termination-log
+ echo "Telegraf is not running" > /dev/write-to-traces  # this file is tailed and printed to stdout
```

Comment:
If you write to stderr for our container, shouldn't we already be picking that up ?

Reply:
Liveness probe output is not written to container stdout :(

Reply:
ok makes sense. Can you also include the changes for tailing this file ?

Reply:
Tailing the file is included, see main.sh line 630

Reply:
so this will go to our stdout and we will pick it up, which looks good. But only thing is if cusotmer turns on kube-system log, it will also end up in customer's workspace. Rather if you just tail this file thru fluentbit, and route to our telemetry it will be quiet for customers. But either way is fine.

Reply:
done, the liveness probe failure message is no longer sent to the container stdout